docker/production/entrypoint.sh

```diff
@@ -9,6 +9,13 @@ nginx -v
 export GIN_MODE=release
 /opt/service/ivory &
 
+# insert base url
+envsubst '\$BASE_URL' < /etc/nginx/nginx.conf > "/etc/nginx/nginx.conf.tmp" && mv "/etc/nginx/nginx.conf.tmp" "/etc/nginx/nginx.conf"
```

Comment:
tbh, I don't like this solution, I was checking as well how to do it better, the main problem that nginx doesn't work with env variables, I want to check if there is more elegant way to do this. I'm even thinking changing nginx :joy:

Reply:
Yeah. I understand. It's a little bit awkward, but even [Nginx themself use `envsubst` in their images](https://github.com/nginxinc/docker-nginx/blob/master/entrypoint/20-envsubst-on-templates.sh).
Anyway, I'm also wondering if there is a better solution🤔.

Reply:
and there is one more problem that even when we have relative path in html, it doesn't work properly with trailing slash, to work it correctly we need to add `<base>` element with this `url`. I'm trying to do it right now will see. I will share the changes with a bit later today probably.

Reply:
> it doesn't work properly with trailing slash

I don't get it, but I guess I'll see it in the changes👍.

Reply:
this is a proper solution I believe, I've decided to get rid of nginx and use go server, I wanted it to do long time ago, because it will help to ease setup of TLS and usage of http/2. You can check it here https://github.com/veegres/ivory/pull/530 I will release it under 1.3.4. 

P.S. Thanks for you effort, your comments were useful, it helped me to know something new, hope you as well :) 
P.S.S I will close this PR